product_packaging_dimension/models/product_packaging.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    max_weight = fields.Float("Weight (kg)")
+    # lngth IS NOT A TYPO https://github.com/odoo/odoo/issues/41353
+    lngth = fields.Integer("Length (mm)", help="length in millimeters")
```

Comment:
```suggestion
    pack_length = fields.Integer("Length (mm)", help="length in millimeters")
```
and idem for other variable to be coherent

Reply:
Here we don't want to use `pack_height` or `pack_width` because we want to redefine the fields from `delivery` module. So it makes more sense to simply use `lngth` instead of `length` as explained in the linked issue.

Reply:
what about this https://github.com/OCA/OCB/commit/25616b9f2bbca3f03dbde1dc81a57986241c1aab ?